packages/tools/kadena-cli/src/devnet/utils/docker.ts

```diff
@@ -2,12 +2,12 @@ import chalk from 'chalk';
 import { execSync } from 'child_process';
 import type { IDevnetsCreateOptions } from './devnetHelpers.js';
 
-const volumePrefix = 'kadena_';
-const containerDataFolder = '/data';
-const containerPactFolder = '/pact-cli';
-const containerPactFolderPermissions = 'ro';
-const chainwebNodeApiPort = '8080';
-const devnetImageName = 'kadena/devnet';
+const volumePrefix: string = 'kadena_';
+const containerDataFolder: string = '/data';
+const containerPactFolder: string = '/pact-cli';
+const containerPactFolderPermissions: string = 'ro';
+const chainwebNodeApiPort: string = '8080';
+const devnetImageName: string = 'kadena/devnet';
```

Comment:
is this a eslint rule or something? because this adds nothing for typescript and to me feels very redundant.